src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -82,12 +94,9 @@ object DepositHandler {
       case InvalidDepositException(_, msg, cause) =>
         log.error(s"[$id] Invalid deposit", cause)
         DepositProperties.set(id, "INVALID", msg, lookInTempFirst = true)
-      case FailedDepositException(_, msg, cause) =>
-        log.error(s"[$id] Failed deposit", cause)
-        DepositProperties.set(id, "FAILED", msg, lookInTempFirst = true)
-      case NonFatal(e)  =>
-        log.error(s"[$id] Unexpected failure in deposit", e)
-        DepositProperties.set(id, "FAILED", "Unexpected failure in deposit", lookInTempFirst = true)
+      case NonFatal(e) =>
+        log.error(s"[$id] Internal failure in deposit", e)
```

Comment:
"... in deposit *service*" to make it more explicit, otherwise it half sounds like we are still blaming the client.